pages/index.vue

```diff
@@ -1,7 +1,57 @@
 <template>
   <article>
+    <OneXPBanner
+      copy="🎉 Qiskit is getting a new documentation and learning experience on IBM Quantum!"
+      :cta="{
+        label: 'Learn more',
+        segment: {
+          cta: 'learn-more-1xp',
+          location: '1xp-banner',
+        },
+        url: '#new-experience',
+      }"
+    />
     <HomeLeadSpace :version="qiskitVersion" />
     <HomeCapabilities />
+    <OneXPPageSection
+      id="new-experience"
+      :features="[
```

Comment:
non-blocking

it'd be nice to have this array of features defined below vs inline... might be cleaner

Reply:
I think that this is a personal preference. IMO, since only this component will use this data, it's unnecessary to declare a variable. Also, it would move the data away from the template unnecessarily, making it harder to reference.